cli/command/plugin/list.go

```diff
@@ -17,6 +21,12 @@ type listOptions struct {
 	filter  opts.FilterOpt
 }
 
+type byName []*types.Plugin
+
+func (n byName) Len() int           { return len(n) }
+func (n byName) Swap(i, j int)      { n[i], n[j] = n[j], n[i] }
+func (n byName) Less(i, j int) bool { return sortorder.NaturalLess(n[i].Name, n[j].Name) }
```

Comment:
I'm not sure we need this lib. Should be able to compare strings directly.

Reply:
Ok, makes sense.

Reply:
Yes; it was added for natural sort